NewTutorials/Tutorial/TutorialCasesViewController.swift

```diff
@@ -50,3 +50,40 @@ class TutorialCasesViewController: UITableViewController {
         props.rows[indexPath.row].action()
     }
 }
+
+class PlayerViewControllerWrapper: UIViewController {
+    struct Props {
+        var isLoading = false
+    }
+    
+    var props = Props() {
+        didSet {
+            view.setNeedsLayout()
+        }
+    }
+    let playerViewController = PlayerViewController()
```

Comment:
Make it `private` - this controller have to work on `PlayerViewController` himself.